examples/java-from-typescript-type/index.ts

```diff
@@ -0,0 +1,45 @@
+import { JavaGenerator } from '../../src';
+import * as path from 'path';
+import * as fs from 'fs';
+
+const generator = new JavaGenerator();
+const generatorWithOptions = new JavaGenerator(
+  {
+    processorOptions: {
+      typescript: {
+        uniqueNames: true,
+        compilerOptions: {
+          strictNullChecks: true
+        }
+      }
+    }
+  }
+);
```

Comment:
Not sure I truly like having two generations within the same example 🤔 

Would it not make more sense to have two instead?

Reply:
I was confused on this tbh. The process and final outcome of both of them are the same, they only differ by the input `options` parameter.

I can split into two use cases if it's easier for people to understand.